hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -726,7 +726,7 @@ private void assignInserts(WorkloadProfile profile) {
               insertRecordsPerBucket = config.getParquetMaxFileSize() / averageRecordSize;
             }
 
-            int insertBuckets = (int) Math.max(totalUnassignedInserts / insertRecordsPerBucket, 1L);
+            int insertBuckets = (int) Math.max(Math.ceil((1.0) * totalUnassignedInserts / insertRecordsPerBucket), 1L);
```

Comment:
did you intend to do `Math.ceil((1.0 * totalUnassignedInserts) / insertRecordsPerBucket)` instead? Also with ceil, do we still need Math.max?

Reply:
Actually, we don't need Math.max, ceil will always give minimum 1 and we already have an if condition for > 0..